src/main/resources/anniv5Resources/localization/eng/goddessofexplosionspack/Cardstrings.json

```diff
@@ -0,0 +1,51 @@
+{
+  "${ModID}:ChainReaction": {
+    "NAME": "Chain Reaction",
+    "DESCRIPTION": "Deal !D! damage. NL Apply !M! ${ModID}:Fallout."
```

Comment:
Since you only use this once, might be worth it to jsut type it out instead of making it a keyword.